packages/svelte-ux/src/lib/components/TextField.svelte

```diff
@@ -366,7 +367,7 @@
                 {max}
                 {step}
                 {actions}
-                bind:inputEl
+                bind:inputEl={null, (el) => (inputEl = el ?? null)}
```

Comment:
this is using function bindings as defined [here](https://svelte.dev/docs/svelte/bind#Function-bindings). The problem was that svelte thought we wanted to provide the value from TextField to Input, and since `inputEl` now may be a TextArea it was not happy with that since there's missing properties. This ensures that data flow is only one way out of the Input and into the TextField variable. I would have put a comment in code but there's not a good way to do that here.

Reply:
Thanks @hahn-kev.  As of right now Svelte UX supports Svelte 3-5 (and will be for 1.x, but will be Svelte 5 only).  I plan to release Svelte UX 1.0 VERY soon (maybe later this week) and have a [PR](https://github.com/techniq/svelte-ux/pull/530) that has started the migration to Svelte 5, but there is a lot of regression testing and refinements that need done.

With that said, we won't be able to use Svelte 5 function bindings yet (the docs are running Svelte 5 so it succeeds).

Here's my [rough plan](https://github.com/techniq/svelte-ux/pull/530#issuecomment-2582849134) for Svelte UX 2.x (and others).  Once we start the `next` branch I would love to include this, and plan to have incremental releases.

Reply:
Ahh ok, I didn't realize that was a svelte 5 feature. I've changed this to do the same thing but an object to act as a wrapper which lets us define a setter without a getter.

Reply:
Nice technique!  Care to run `pnpm format` to fix the prettier lint issue and I'll merge / release.

Reply:
Also, could you add an example or a way to validate the change.  I don't have any component tests at the moment, just some unit tests and a lot of examples for visual/manual validation

Reply:
fixed the lint issues. FYI maybe this is because I'm on windows, but when I ran format it modified 486 files, in addition to the TextField file. I only checked in that one. That might trip up others though, I think it might be a BOM issue, but it's hard to tell what the changes are.

Reply:
Alright there's now also an example, here's a screenshot of it, I figured I should include a toggle between single and multi-line since that's what this PR was mainly for.
![image](https://github.com/user-attachments/assets/ec1fcaa2-a2a2-47ca-8ce6-180a294bc812)